python/pyspark/pandas/internal.py

```diff
@@ -120,13 +252,13 @@ class InternalFrame(object):
 
     * `data_spark_columns` represents non-indexing Spark columns
 
-    * `data_dtypes` represents external non-indexing dtypes
+    * `data_fields` represents non-indexing field metadata
```

Comment:
I'm wondering if `field metadata` and `Return the metadata for the field` are confusing for people out of context.

Reply:
Makes sense. I changed the wording.